### PR TITLE
Update the document to highlight some words

### PR DESCRIPTION
**Pull request by xichengliudui**
Signed-off-by: xichengliudui <liuduidui@beyondcent.com>

**What this PR does / why we need it**:
Update the document to highlight some words

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: docs/components.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ functionality is not provided by kubernetes itself.
 We will try to leverage as much of Kubernetes regarding to mounting and preparing images for VMI.
 However, `virt-handler` may provide a plugin mechanism to allow storage mounting and setup from the host, if the KubeVirt requirements do not fit into the Kubernetes storage scenarios.
 
-Since host side preparation of storage may not be enough, a cluster-wide [Storage Controller](###storage-controller) can be used to prepare storage.
```
Comment (xichengliudui) [inline docs/components.md]: The Storage Controller link has expired, so I changed it to an absolute link. the Network Controller also fails.

Comment (slintes) [reply docs/components.md]: according to [1] a single hash should work for relative links, did you try that?

[1] https://gist.github.com/asabaylus/3071099

Comment (xichengliudui) [reply docs/components.md]: Sorry to keep you waiting, I have updated it.

**Pull request closed without merging**